README.markdown

```diff
@@ -459,6 +459,50 @@ var diameter: Double {
 }
 ```
 
+When the computational overhead is small, a computed property is preferred to a function that takes no parameters and returns a value. Use a function to signal to the caller that the computation is non-trivial. E.g. 
+
+**Preferred**
+```swift
+var urlString: String?  {
+  var components = URLComponents()
+  components.scheme = "blueapron"
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var func toTokens() -> [String]  {
+    let scanner = Scanner()
+    var tokens = [String]()
+     while !(scanner.isAtEnd) {
+        // Complex parsing of a potentially long string
+    }
+    return tokens
+}
+```
+
+**Not preferred**
+```swift
+func toURL() -> URL {
+  var components = URLComponents()
+  components.scheme = "blueapron"
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var tokens: [String]  {
+    let scanner = Scanner()
+    var tokens = [String]()
+     while !(scanner.isAtEnd) {
```

Comment:
This one too!

Reply:
oops fixing these, you have a great future ahead of you as a human linter